ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/Claims.scala

```diff
@@ -53,6 +53,11 @@ final case class ClaimReadAsParty(name: Ref.Party) extends Claim
   * They also optionally specify an expiration epoch time that statically specifies the
   * time on or after which the token will no longer be considered valid by the Ledger API.
   *
+  * @param claims         List of [[Claim]]s describing the authorization this object describes (see table below).
```

Comment:
The scaladoc seems to be in the middle of the rest of the doc comments. Let's move this part to the end?